Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'jbuilder', '~> 2.0' # DSL for building JSON view template
 gem 'haml-rails' # HTML template language, used instead of ERB
 gem 'bootsnap', require: false # Makes rails boot faster via caching
 gem 'figaro' # easy access to ENV variables. Deprecated.
+gem 'oj'
```

Comment:
Please move this the 'http and api tools' section, and add a comment that describes the gem. With a name like `oj`, its role in the app will be opaque otherwise.